From 3f0e4d78f347d9bb8ebe0fc7d49bcfeb1f84384e Mon Sep 17 00:00:00 2001
From: Hanlu Li <heuleehanlu@gmail.com>
Date: Mon, 7 Apr 2025 10:10:00 +0800
Subject: [PATCH 1029/1058] LATX, doc: update README with Arch package
 requirements

Add required packages for building on Arch Linux to the README

Signed-off-by: Hanlu Li <heuleehanlu@gmail.com>
---
 README.en.rst | 14 ++++++++++++++
 README.rst    |  3 +++
 2 files changed, 17 insertions(+)

diff --git a/README.en.rst b/README.en.rst
index e98450f340..5cf4f0e3c9 100644
--- a/README.en.rst
+++ b/README.en.rst
@@ -153,6 +153,20 @@ automation, and testing:
 
     - Contributor for adaptation to new-world support
 
+Build
+=====
+
+.. code-block:: text
+# debian sid
+apt install -y git ninja-build libssl-dev libc6 gcc g++ pkg-config libglib2.0-dev libdrm-dev lsb-release make python3-setuptools
+git clone --depth=1 --recursive https://github.com/lat-opensource/lat
+cd lat/latxbuild
+./build-release.sh
+
+.. code-block:: text
+# Arch Linux
+pacman -S --noconfirm ninja gcc pkgconf python3 python-setuptools openssl-static openssl
+
 Future Plans (TODO)
 ===============
 
diff --git a/README.rst b/README.rst
index 4a720b71e5..db85b74aac 100644
--- a/README.rst
+++ b/README.rst
@@ -148,6 +148,9 @@ LATX 基于 QEMU 6 版本开发并进行了深度优化，性能相比原生 QEM
     ./build-release.sh
 
 
+.. code-block:: text
+# Arch Linux
+pacman -S --noconfirm ninja gcc pkgconf python3 python-setuptools openssl-static openssl
 
 未来规划（TODO）
 ===============
-- 
2.49.0

